### ISSUE TITLE
Broken Set Role scenario

### ISSUE THREAD

**Issue by HonzaBejvl** [IvanJosipovic/BlazorApplicationInsights#37]
#32 broke SetRoleScenario. 

In my opinion, it is bad practice to use IJSRuntime outside of OnAfterRenderAsync. It blocks blazor-server scenarios and adds complexity to WebAssemblyPrerendered mode. 

#36 Please take is PR just as a proposal/idea of how to go around this problem.

It allows a consumer to configure OnBlazorInsightsInit callback that is executed during the first OnAfterRenderAsync of ApplicationInsightsComponent

**Comment by HonzaBejvl** [comment]
Configuration then could look like that: 
```
builder.Services.AddBlazorApplicationInsights(async applicationInsights =>
            {
                var telemetryItem = new TelemetryItem()
                {
                    Tags = new Dictionary<string, object>()
                {
                    { "ai.cloud.role", "SPA" },
                    { "ai.cloud.roleInstance", "Blazor Wasm" },
                }
                };

                await applicationInsights.AddTelemetryInitializer(telemetryItem);
            });
```

**Comment by IvanJosipovic** [comment]
Hey,

I really appreciate your feedback and both PRs. I just released your changes in [0.7.0](https://github.com/IvanJosipovic/BlazorApplicationInsights/releases/tag/v0.7.0).

Thanks!
